tests/util.spec.coffee

```diff
@@ -110,6 +110,48 @@ describe 'Pine option merging', ->
 						application: {}
 						build: {}
 
+	it 'combines $expand params for expand options that are arrays of objects', ->
+		result = mergePineOptions
+			expand: [
+				device: $select: [ 'id' ]
+			]
+		,
+			expand: [
+				device: $expand: [ 'build' ]
+			,
+				application: $expand: [ 'release' ]
+			]
+
+		m.chai.expect(result).to.deep.equal
+			expand:
+				application:
+					$expand: [ 'release' ]
+				device:
+					$select: [ 'id' ]
+					$expand: [ 'build' ]
+
+	it 'combines $expand params for expand options that are arrays of objects with multiple keys', ->
+		result = mergePineOptions
+			expand: [
+				device: $select: [ 'id' ]
+				application: $expand: [ 'user' ]
+			]
+		,
+			expand: [
+				device: $expand: [ 'build' ]
+				application: $expand: [ 'release' ]
```

Comment:
Would it make sense to add a test doing `application: $expand: [ 'user' ]` & `application: $expand: 'release'`?

Reply:
Yeah we could do